rules/concepts.md

```diff
@@ -48,6 +48,154 @@ In some cases, it is useful to have a rule that is only manually triggered.
 Sometimes there is advantage in a rule that calls another rule.
 In both cases, the rule has no triggers and no conditions.
 
+## Triggers
 
+Now that we know the concept of rule triggers, let's look at them in more depth.
+Triggers define those events that, when they occur, causes the rule to run.
 
+There are the categories of rules that can be used to trigger a rule:
 
+| Event       | Description                                                                                                                  |
+|-------------|------------------------------------------------------------------------------------------------------------------------------|
+| **Items**   | Commands, updates, and changes on an individual Item.                                                                        |
```

Comment:
> on an individual Item's state.

There are other changes that could occur on an Item that don't generate events like creation, deletion, metadata added/removed, etc. I don't want to leave any doubt here that we are only referencing state.